pkg/setting/setting.go

```diff
@@ -1579,6 +1582,17 @@ func readAuthSettings(iniFile *ini.File, cfg *Cfg) (err error) {
 	// Azure Auth
 	AzureAuthEnabled = auth.Key("azure_auth_enabled").MustBool(false)
 	cfg.AzureAuthEnabled = AzureAuthEnabled
+
+	// ID response header
+	cfg.IDResponseHeaderEnabled = auth.Key("id_response_header_enabled").MustBool(false)
+	cfg.IDResponseHeaderPrefix = auth.Key("id_response_header_prefix").MustString("X-Grafana-")
+
+	idHeaderNamespaces := strings.Split(auth.Key("id_response_header_namespaces").MustString(""), " ")
```

Comment:
Nit. We can use `util.SplitString` to be consistent with how we handle lists from the ini file (supports json, or `,` and ` ` separated values )